editor/src/core/shared/uid-utils.spec.ts

```diff
@@ -86,3 +86,18 @@ describe('updateHighlightBounds', () => {
     expect(actualResult).toEqual(expectedResult)
   })
 })
+
+describe('nextTestUID', () => {
+  it("increments a string's last character", async () => {
+    expect(nextTestUID('aaa')).toEqual('aab')
+    expect(nextTestUID('aab')).toEqual('aac')
+    expect(nextTestUID('aba')).toEqual('abb')
+    expect(nextTestUID('aaz')).toEqual('aba')
+    expect(nextTestUID('azz')).toEqual('baa')
+    expect(nextTestUID('a9a')).toEqual('aab')
+    expect(nextTestUID('aa9')).toEqual('aab')
+  })
+  it('throws if it cannot generate anymore', async () => {
+    expect(() => nextTestUID('zzz')).toThrow('Unable to generate a UID')
```

Comment:
at this point I think we should roll over to 4 character UIDs, right?

Reply:
we sure could, this was meant to be 1:1 with the previous implementation which only goes as far as 3 chars, but no problem in extending it

Reply:
oh keeping the existing behavior is a good idea!